### ISSUE TITLE
site permission requested every time I open an .md noe

### ISSUE THREAD

**Issue by notuntoward** [daeh/zotero-markdb-connect#136]
Every time I run `Open Note in Obsidian`, I see the popup below.  Is there a way I can stop this?

![image](https://github.com/user-attachments/assets/a84a229a-3846-4b11-a6bd-bc5240aa1249)

I'm on Windows 7, Zotero 7, and am using the version of [zotero-markdb-connect](https://github.com/daeh/zotero-markdb-connect), as of of 8/17/24

**Comment by daeh** [comment]
Yeah, that's the case on other platforms too. I have been presuming that it's a Zotero feature in beta state - I presume that the stable version of Zotero will remember link permissions. (If someone knows otherwise, please let me know). So hopefully this will resolve itself upstream, but if it doesn't then I'll definitely look for a solution.

**Comment by tinysun212** [comment]
I'm using only one Obsidian Vault, and when I select `System's Default Editor` instead of `Obsidian` for `Open Markdown Files using:` in the MarkDB-Connect Setting, the popup doesn't appear.

**Comment by daeh** [comment]
For the moment, you can turn off the security feature that's raising the notification by setting `security.external_protocol_requires_permission` to `false`.

Zotero settings > Advanced tab > Config Editor" > Accept Risk and Continue > search for `security.external_protocol_requires_permission` > double click the string to change value to `false`.

**Comment by daeh** [comment]
The new protocol for Zotero 9 is detailed in the README.
